From 36930ab88ee579df292a7b36f84d426a12bba03e Mon Sep 17 00:00:00 2001
From: Evgeni Ermakov <evgeni_ermakov@Mail.ru>
Date: Sat, 23 Oct 2021 21:12:30 +0300
Subject: [PATCH] feat: add 11 and 12 row

---
 firstFile.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/firstFile.txt b/firstFile.txt
index f00c965..3574519 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -8,3 +8,5 @@
 8
 9
 10
+11
+12
\ No newline at end of file
-- 
2.33.0.windows.2

